pysndfx/dsp.py

```diff
@@ -83,6 +84,59 @@ def compand(self, attack=0.05, decay=0.5):
         raise NotImplemented()
         return self
 
+    def sinc(self, hpfreq=None, lpfreq=None,
```

Comment:
Please choose longer and clearer names for arguments (let's assume the user is _not_ familiar with sox's CLI).

Reply:
What would you suggest? `high_pass_freq`?

Reply:
`high_pass_frequency` and `low_pass_frequency` are probably best.

I should probably not have picked this also: https://github.com/carlthome/python-audio-effects/blame/68e1f3a5713afb6ecf02d7abf3f287ab309ad37e/pysndfx/dsp.py#L202  🐌

Reply:
Ok for the args. I'll probably update the pull request this week end.

Why do you think that wet_only is wrong?

Reply:
`wet_only` is fine, but `hf_damping` should have been written out, IMO.